contributing/publishing.md

```diff
@@ -57,7 +63,7 @@ Before approving the release to the marketplace, download the vsix files from th
 
 Alternatively, you can download the files using the [gh cli](https://cli.github.com/) and then upload them all at once. Replace `v57.3.0` with the tag name for the release that you are testing, and to whatever download directory you would like. Additionally, `code` can be replaced by `code-insiders`.
 
-`> gh release download v57.3.0 --dir ~/Downloads/v57.3.0 --pattern '*.vscode'`  
+`> gh release download v57.3.0 --dir ~/Downloads/v57.3.0 --pattern '*.vscode'`
```

Comment:
```suggestion
`> gh release download v57.3.0 --dir ~/Downloads/v57.3.0 --pattern '*.vsix'`
```

Reply:
@jeffb-sfdc I noticed that I had the wrong pattern here during my release testing!